### ISSUE TITLE
YOLOv5-v6.0 Initialisation ISSUE WITH PYTHON3: 

### ISSUE THREAD

**Issue by Symbadian** [pytorch/pytorch#84110]
### 🐛 Describe the bug

Hi All,

Thank you for the great works on the new version of YOLOv5-v6.0, your efforts does not go unnoticed.

However, I am experiencing an error with the new build when trying to execute the initialisation commands. I followed @glenns-Jocher's instructions to the best of my understanding via the installations of the requirements and then from there it should be smooth sailing. I am sure that this is not one of the attributes that should follow when processing the training sessions as stated in the instructions.

WHAT I DID:
1. Deleted the old yolov5 environment and started all over to create space for the new build
2. Re-installed all of the packages as per the requirements.txt file
3. Ran the initialisation code

The error is extremely persistent and I tried googling to solve this but no matter what I do this is my results. I am at the stage where I am going to wipe my NEW MAC 2021 MAX 32cores M1 Silicon Chip to get rid of this annoying issue.

*****Illegal instruction: 4*******
![Screenshot 2022-08-26 at 08 52 14](https://user-images.githubusercontent.com/50293021/186851711-5dbc1237-3070-453b-838e-323cf32f8514.png)

PACKAGES:
`
 symbadian$ pip3 list
Package                      Version
---------------------------- --------------------
absl-py                      1.2.0
anyio                        3.6.1
appnope                      0.1.3
arel                         0.2.0
asttokens                    2.0.8
astunparse                   1.6.3
async-asgi-testclient        1.4.11
backcall                     0.2.0
bidict                       0.22.0
CacheControl                 0.12.11
cachetools                   5.2.0
cachy                        0.3.0
certifi                      2022.6.15
charset-normalizer           2.1.1
cleo                         0.8.1
click                        8.1.3
clikit                       0.6.2
coremltools                  6.0b2
crashtest                    0.3.1
cycler                       0.11.0
decorator                    5.1.1
distlib                      0.3.5
executing                    0.9.1
fastapi                      0.80.0
filelock                     3.8.0
flatbuffers                  1.12
fonttools                    4.36.0
gast                         0.4.0
google-auth                  2.11.0
google-auth-oauthlib         0.4.6
google-pasta                 0.2.0
grpcio                       1.47.0
h11                          0.13.0
h5py                         3.7.0
html5lib                     1.1
httptools                    0.4.0
idna                         3.3
imageio                      2.21.1
imgaug                       0.4.0
ipython                      8.4.0
jedi                         0.18.1
Jinja2                       3.1.2
jsonpatch                    1.32
jsonpointer                  2.3
keras                        2.9.0
Keras-Preprocessing          1.1.2
keyring                      23.8.2
kiwisolver                   1.4.4
libclang                     14.0.6
lockfile                     0.12.2
Markdown                     3.4.1
MarkupSafe                   2.1.1
matplotlib                   3.5.3
matplotlib-inline            0.1.6
mpmath                       1.2.1
msgpack                      1.0.4
multidict                    6.0.2
natsort                      8.1.0
networkx                     2.8.6
numpy                        1.23.2
oauthlib                     3.2.0
opencv-python                4.6.0.66
opt-einsum                   3.3.0
packaging                    20.9
pafy                         0.5.5
pandas                       1.3.5
parso                        0.8.3
pastel                       0.2.1
pexpect                      4.8.0
pickleshare                  0.7.5
Pillow                       9.2.0
pip                          22.2.2
pkginfo                      1.8.3
platformdirs                 2.5.2
poetry                       1.1.15
poetry-core                  1.0.8
prompt-toolkit               3.0.30
protobuf                     3.19.4
psutil                       5.9.1
PTable                       0.9.2
ptyprocess                   0.7.0
pure-eval                    0.2.2
pyasn1                       0.4.8
pyasn1-modules               0.2.8
pydantic                     1.9.2
pydicom                      2.3.0
Pygments                     2.13.0
pylev                        1.4.0
pynrrd                       0.4.3
pyparsing                    3.0.9
python-dateutil              2.8.2
python-dotenv                0.20.0
python-json-logger           2.0.4
python-magic                 0.4.27
pytz                         2022.2.1
PyWavelets                   1.3.0
PyYAML                       6.0
requests                     2.28.1
requests-oauthlib            1.3.1
requests-toolbelt            0.9.1
rsa                          4.9
scikit-image                 0.19.3
scikit-video                 1.1.11
scipy                        1.9.0
seaborn                      0.11.2
setuptools                   63.2.0
Shapely                      1.8.4
shellingham                  1.5.0
SimpleITK                    2.1.1.2
six                          1.16.0
sniffio                      1.2.0
stack-data                   0.4.0
starlette                    0.19.1
stringcase                   1.2.0
supervisely                  6.58.4
sympy                        1.10.1
tensorboard                  2.9.1
tensorboard-data-server      0.6.1
tensorboard-plugin-wit       1.8.1
tensorflow                   2.9.1
tensorflow-estimator         2.9.0
tensorflow-io-gcs-filesystem 0.26.0
tensorflow-macos             2.9.2
termcolor                    1.1.0
thop                         0.1.1.post2207130030
tifffile                     2022.8.12
tomlkit                      0.11.4
torch                        1.12.1
torchaudio                   0.12.1
torchvision                  0.13.1
tqdm                         4.64.0
traitlets                    5.3.0
trimesh                      3.14.0
typing_extensions            4.3.0
urllib3                      1.26.11
uvicorn                      0.18.3
uvloop                       0.16.0
varname                      0.9.0
virtualenv                   20.16.3
watchfiles                   0.16.1
watchgod                     0.6
wcwidth                      0.2.5
webencodings                 0.5.1
websockets                   10.3
Werkzeug                     2.2.2
wheel                        0.37.1
wrapt                        1.14.1
youtube-dl                   2021.12.17
`
 

### Versions

`
symbadian$ python3 train.py --img 640 --batch 16 --epochs 3 --data coco128.yaml  --weights yolov5s.pt
train: weights=yolov5s.pt, cfg=, data=coco128.yaml, hyp=data/hyps/hyp.scratch-low.yaml, epochs=3, batch_size=16, imgsz=640, rect=False, resume=False, nosave=False, noval=False, noautoanchor=False, noplots=False, evolve=None, bucket=, cache=None, image_weights=False, device=, multi_scale=False, single_cls=False, optimizer=SGD, sync_bn=False, workers=8, project=runs/train, name=exp, exist_ok=False, quad=False, cos_lr=False, label_smoothing=0.0, patience=100, freeze=[0], save_period=-1, seed=0, local_rank=-1, entity=None, upload_dataset=False, bbox_interval=-1, artifact_alias=latest
github: skipping check (not a git repository), for updates see https://github.com/ultralytics/yolov5
YOLOv5 🚀 2022-8-17 Python-3.10.6 torch-1.12.1 CPU

hyperparameters: lr0=0.01, lrf=0.01, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=0.05, cls=0.5, cls_pw=1.0, obj=1.0, obj_pw=1.0, iou_t=0.2, anchor_t=4.0, fl_gamma=0.0, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, mosaic=1.0, mixup=0.0, copy_paste=0.0
Weights & Biases: run 'pip install wandb' to automatically track and visualize YOLOv5 🚀 runs in Weights & Biases
ClearML: run 'pip install clearml' to automatically track, visualize and remotely train YOLOv5 🚀 in ClearML
TensorBoard: Start with 'tensorboard --logdir runs/train', view at http://localhost:6006/
Illegal instruction: 4
`

**Comment by ezyang** [comment]
Wrong repo. Send this report to the YOLOv5 repo.
